monitor.py

```diff
@@ -6,18 +6,20 @@
 import requests
 from datetime import datetime
 import xml.etree.ElementTree as ET
+import monitoring_config
 
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 PROW_URL = ""
-PROW_VIEW_URL = "https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs"
 final_job_list=[]
-RELEASE_URL = "https://ppc64le.ocp.releases.ci.openshift.org/releasestream/4-stable-ppc64le/release/"
+prow_view_url = monitoring_config.PROW_VIEW_URL
```

Comment:
This is not required, wherever the variables are used, access it like "constants.PROW_VIEW_URL"